src/Campaigns/Actions/LoadCampaignDetailsAssets.php

```diff
@@ -14,30 +12,33 @@ class LoadCampaignDetailsAssets
     /**
      * @unreleased
      */
-    public function __invoke(Campaign $campaign)
+    public function __invoke()
     {
         $handleName = 'givewp-admin-campaign-details';
-        $asset = ScriptAsset::get(GIVE_PLUGIN_DIR . 'assets/dist/js/give-admin-campaign-details.asset.php');
+        $scriptAsset = ScriptAsset::get(GIVE_PLUGIN_DIR . 'build/campaignDetails.asset.php');
+
+        wp_enqueue_editor();
 
         wp_register_script(
             $handleName,
-            GIVE_PLUGIN_URL . 'assets/dist/js/give-admin-campaign-details.js',
-            $asset['dependencies'],
-            $asset['version'],
+            GIVE_PLUGIN_URL . 'build/campaignDetails.js',
+            $scriptAsset['dependencies'],
+            $scriptAsset['version'],
             true
         );
 
         wp_localize_script($handleName, 'GiveCampaignDetails',
             [
-                'apiRoot' => esc_url_raw(rest_url('give-api/v2/campaigns')),
-                'apiNonce' => wp_create_nonce('wp_rest'),
                 'adminUrl' => admin_url(),
-                'pluginUrl' => GIVE_PLUGIN_URL,
-                'campaign' => (new CampaignDetailsPage($campaign))->exports(),
+                'currency' => give_get_currency()
             ]
         );
 
         wp_enqueue_script($handleName);
         wp_enqueue_style('givewp-design-system-foundation');
+        wp_enqueue_style(
+            $handleName,
+            GIVE_PLUGIN_URL . 'build/style-campaignDetails.css'
```

Comment:
We need to change this line to this `GIVE_PLUGIN_URL . 'build/campaignDetails.css'` so we can properly load the CSS files for the Form List Table in the Forms tab.

But it will introduce a new problem that prevents the details page styles from being loaded as you can see in this screenshot:

![image](https://github.com/user-attachments/assets/46b2d2dd-eeea-446f-b649-d7bbaf8bbf20)

So to fix this one you need to implement the change that I mentioned in this comment: https://github.com/impress-org/givewp/pull/7547/files#r1777705915

After that, we'll be able to have all styles being loaded like you can check on this screenshot: 

![image](https://github.com/user-attachments/assets/7f948c94-7a60-4aab-b356-72cb8f08f6b5)